generators/new-wp-plugin/index.js

```diff
@@ -92,11 +91,33 @@ module.exports = generators.Base.extend({
       regex: 'lean-p',
       replacement: this.name
     }, settings) );
+    console.log( chalk.green('Completed!\n') );
   },
 
   install: function() {
     process.chdir( this.PLUGIN_FOLDER );
 
-    this.spawnCommandSync('composer', ['update']);
+    if ( fs.existsSync('./vendor') ) {
+      console.log( chalk.red('Composer has alredy been installed\n') );
+    } else {
+      this.spawnCommandSync('composer', ['install']);
```

Comment:
They do the same thing but not as we should expect. Some scripts like Yoast or ACF relies on the fires of the composer events like `post-install`, `post-update`. There is a reason why install exist same as update.

Install should be always be always running at the first time of the project after that we only should use update.

That's why each composer has this flow in order to allow set up on install an not on update, please read the docs about this if you have more questions.